test/compflags/bradc/help/userhelp.good

```diff
@@ -1,5 +1,27 @@
 Usage: chpl [flags] [source files]
 
+Primary Configuration Options:
+      --home <path>                   Path to Chapel's home directory
```

Comment:
Yeah I commented on that in the description of the PR. Let's do that in a separate PR, but I agree that it should be consistent with the man-page order.